pkg/beholder/utils/common.go

```diff
@@ -0,0 +1,98 @@
+package utils
```

Comment:
Why a new package? Can these just be part of `package beholder`?

Reply:
moved them into `package beholder`